docs/content/install/install-on-kubernetes.md

```diff
@@ -223,3 +223,34 @@ In order to instruct athens to fetch and use the secret, `netrc.enabled` flag mu
 ```console
 helm install gomods/athens-proxy -n athens --namespace athens --set netrc.enabled=true
 ```
+
+### gitconfig support
+
+A [gitconfig](https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration) file can be shared as a secret to allow
+the access to modules in private git repositories. For example, you can configure access to private repositories via HTTPS
+using personal access tokens on GitHub, GitLab and other git services.
```

Comment:
@hypnoglow would you mind mentioning here that this approach uses basic authentication?

Reply:
Strictly speaking, if there would be a possibility to inject ssh keys into a container,  then this approach would be not only about basic auth - we could use git config to rewrite https:// urls to ssh:// urls to force cloning via SSH using keys.

But, as long as there is currently no option to pass SSH keys to a container yet, I agree that this is mostly about basic auth (or token auth on GitHub/GitLab, that is implemented on top of basic auth protocol).

Reply:
Added sentence: `Basically, this approach uses basic authentication.`